specs/electra/validator.md

```diff
@@ -66,7 +66,12 @@ def compute_on_chain_aggregate(network_aggregates: Sequence[Attestation]) -> Att
     committee_flags = [(index in committee_indices) for index in range(0, MAX_COMMITTEES_PER_SLOT)]
     committee_bits = Bitvector[MAX_COMMITTEES_PER_SLOT](committee_flags)
 
-    return Attestation(aggregation_bits, data, committee_bits, signature)
+    return Attestation(
+        aggregation_bits=aggregation_bits,
+        data=data,
+        committee_bits=committee_bits,
+        signature=signature,
+    )
```

Comment:
actually, I don't understand why this fixed the CI error...

without this fix, the result `aggregate.data.target.epoch` is `0` when it should be `2` in some tests.

/cc @protolambda, in case you can share some remerkleable wisdom. 🧙‍♂️

Reply:
not sure either, but I'll just assume there is some subtlety in how remerkleable initializes things